anthropic-java-bedrock/src/main/kotlin/com/anthropic/bedrock/credentials/BedrockCredentials.kt

```diff
@@ -0,0 +1,391 @@
+        /**
+         * The Amazon Bedrock service name used for model inferences.
+         */
+        private const val SERVICE_NAME = "bedrock-runtime"
+
+        /**
+         * The Anthropic version for the Bedrock service.
+         */
+        private const val ANTHROPIC_VERSION = "bedrock-2023-05-31"
+
+        /**
+         * Creates a new builder for configuring and creating a new instance of
+         * [BedrockCredentials].
+         *
+         * @return The new builder.
+         */
+        @JvmStatic fun builder() = Builder()
+
+        /**
+         * Creates new [BedrockCredentials] with credentials resolved from the
+         * environment or configuration files.
+         *
+         * This is a convenience method that is the equivalent of calling:
+         *
+         * ```
+         * BedrockCredentials.builder().fromEnv().build()
+         * ```
+         *
+         * @return The new [BedrockCredentials].
+         *
+         * @throws AnthropicException See [Builder.build] for details.
+         */
+        @JvmStatic fun fromEnv(): BedrockCredentials {
+            return builder().fromEnv().build()
+        }
+
+        /**
+         * Creates a [HttpRequestBody] holding JSON object data.
+         *
+         * @param json The JSON object data to form the request body.
+         *
+         * @return A new request body holding the JSON data. The content type
+         *     will be set to `application/json`.
+         */
+        // NOTE: "internal" visibility allows this method to be used in unit
+        // tests to create test fixtures.
+        @JvmSynthetic
+        internal fun jsonToBody(json: ObjectNode): HttpRequestBody {
+            // TODO: Code copied from "HttpRequestBodies.json", an "internal"
+            //   function. Should that function be made visible for use here?
+            return object : HttpRequestBody {
+                private var cachedBytes: ByteArray? = null
+
+                private fun serialize(): ByteArray {
+                    if (cachedBytes != null) return cachedBytes!!
+
+                    val buffer = ByteArrayOutputStream()
+                    try {
+                        ObjectMapper().writeValue(buffer, json)
+                        cachedBytes = buffer.toByteArray()
+                        return cachedBytes!!
+                    } catch (e: Exception) {
+                        throw AnthropicException("Error writing request", e)
+                    }
+                }
+
+                override fun writeTo(outputStream: OutputStream) {
+                    outputStream.write(serialize())
+                }
+
+                override fun contentType(): String = "application/json"
+
+                override fun contentLength(): Long {
+                    return serialize().size.toLong()
+                }
+
+                override fun repeatable(): Boolean = true
+
+                override fun close() {}
+            }
+        }
```

Comment:
I think you can use `HttpRequestBodies.json` if you do `json(jsonMapper(), JsonValue.fromJsonNode(yourNode))`